apps/front/src/templates/signup-form/hooks/useSignupFormSchema.ts

```diff
@@ -13,7 +13,7 @@ export type FormModel = {
   idSkills: number[];
 };
 
-export const useFormSchema = () => {
+export const useSignupFormSchema = () => {
```

Comment:
As a reminder, we would need to change our convention to pass translated messages directly (and not the translation key)

The change is still waiting to be done on `hublo-front` also by the way